toolchain/check/decl_name_stack.cpp

```diff
@@ -44,7 +44,8 @@ auto DeclNameStack::NameContext::prev_inst_id() -> SemIR::InstId {
 auto DeclNameStack::MakeEmptyNameContext() -> NameContext {
   return NameContext{
       .initial_scope_index = context_->scope_stack().PeekIndex(),
-      .parent_scope_id = context_->scope_stack().PeekNameScopeId()};
+      .parent_scope_id = context_->scope_stack().PeekNameScopeId(),
+      .poisoning_loc_id = SemIR::LocId::None};
```

Comment:
```suggestion
      .parent_scope_id = context_->scope_stack().PeekNameScopeId()};
```

We have defaults for most of this struct's members where possible, so I think it'd be consistent to put the default in the header here.

Reply:
Done.